images/decode.go

```diff
@@ -33,6 +36,15 @@ func Decode(fileName string) (image.Image, error) {
 	return decodeImageData(fb, file)
 }
 
+func DecodeFromB64Uri(b64Uri string) ([]byte, error) {
+	a := strings.Split(b64Uri, ";")
+	if len(a) != 2 || len(a[1]) <= 7 {
+		return nil, errors.New(fmt.Sprintf("b64Uri can not be parsed. b64Uri: %s", b64Uri))
+	}
+	imageB64String := strings.Split(b64Uri, ";")[1][7:]
+	return base64.StdEncoding.DecodeString(imageB64String)
+}
+
```

Comment:
Would you be able to introduce a test for this function? 🙂

Reply:
this function can be removed now, we don't need it anymore, since we don't want to transfer images through the bridge anymore